curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-prototype-properties-to-reduce-duplicate-code.md

```diff
@@ -12,7 +12,7 @@ Since `numLegs` will probably have the same value for all instances of `Bird`, y
 
 This may not be an issue when there are only two instances, but imagine if there are millions of instances. That would be a lot of duplicated variables.
 
-A better way is to use `Bird’s` `prototype`. Properties in the `prototype` are shared among ALL instances of `Bird`. Here's how to add `numLegs` to the `Bird prototype`:
+A better way is to use `Bird`'s `prototype`. Properties in the `prototype` are shared among ALL instances of `Bird`. Here's how to add `numLegs` to the `Bird prototype`:
```

Comment:
```suggestion
A better way is to use the `prototype` of `Bird`. Properties in the `prototype` are shared among ALL instances of `Bird`. Here's how to add `numLegs` to the `Bird prototype`:
```

Reply:
As we can't change what's inside the backticks, having an english grammar feature in other languages could result in weird results and misunderstandings from campers. Also note that the backticks are not actually visible
The result change from `Bird's` to `Bird`'s, visually it's a small difference but allows translators to not bring an english grammar feature to other languages